cubano-webdriver/src/main/java/org/concordion/cubano/driver/web/config/WebDriverConfig.java

```diff
@@ -1,348 +1,127 @@
-        return shouldActivatePlugins();
-    }
-
-    /**
-     * Activate developer plugins - FireFox only browser supported currently and will add FireBug and FirePath.
-     *
-     * @return true or false
-     */
-    public boolean shouldActivatePlugins() {
-        return activatePlugins;
-    }
-
-    /**
-     * Size to set browser window - will default to maximised.
-     *
-     * @return Size in wxh format
-     */
-    public String getBrowserSize() {
-        return browserSize;
-    }
-
-    /**
-     * Default timeout in seconds.
-     *
-     * @return timeout
-     */
-    public int getDefaultTimeout() {
-        return browserDefaultTimeout;
-    }
-
-    /**
-     * Username for remote selenium grid service.
-     *
-     * @return Username
-     */
-    public String getRemoteUserName() {
-        return remoteUserName;
-    }
-
-    /**
-     * Api Key to access a remote selenium grid service.
-     *
-     * @return Api Key
-     */
-    public String getRemoteApiKey() {
-        return remoteApiKey;
-    }
-
-    /**
-     * Proxy should be setup or not.
-     *
-     * @return true or false
-     */
-    public boolean isProxyRequired() {
-        return proxyIsRequired;
-    }
-
-    /**
-     * Proxy host name.
-     *
-     * @return host
-     */
-    public String getProxyHost() {
-        return proxyHost;
-    }
-
-    /**
-     * Proxy username.
-     *
-     * @return username
-     */
-    public String getProxyUser() {
-// TODO: Not sure that proxy user should fail if not provided
-//        if (proxyUsername == null) {
-//            throw new RuntimeException("proxy.username entry must exist in the user.properties file in the root folder");
-//        }
-
-        return proxyUsername;
-    }
-
-    /**
-     * Proxy user.
-     *
-     * @return user
-     */
-    public String getProxyPassword() {
-//  TODO: Not sure that proxy password should fail if not provided
-//        if (proxyPassword == null) {
-//            throw new RuntimeException("proxy.proxypassword entry must exist in the user.properties file in the root folder");
-//        }
-
-        return proxyPassword;
-    }
-
-    /**
-     * @return Proxy bypass (noproxy) addresses, eg: "localhost, 127.0.0.1".
-     */
-    public String getNoProxyList() {
-        return "localhost, 127.0.0.1";
-    }
-
+package org.concordion.cubano.driver.web.config;
+
+import java.util.Map;
+import java.util.Properties;
+import org.concordion.cubano.utils.Config;
+
+/**
+ * Reads and supplies properties from the <code>config.properties</code> file that are required by the framework.
+ * <p>
+ * An optional <code>user.properties</code> file can set user specific values and allow overriding of defaults.
+ * The <code>user.properties</code> file should NEVER be checked into source control.
+ * <p>
+ * This class can be extended by an <code>AppConfig</code> class to provide application specific properties.
+ *
+ * TODO Should this be singleton?

```

Comment:
This class is a singleton so comment can be removed. Just wondering why the base class Config isn't a singleton too?

Reply:
Also want to stop loading properties files for each version of this - something else to discuss with you next time we catch up